documentation/src/docs/table-api.md

```diff
@@ -47,45 +47,45 @@ pravega.tableSourceReaderBuilder()
         .withPravegaConfig(pravegaConfig);
 
 
- // (Option-1) Streaming Source
+// (Option-1) Streaming Source
 StreamExecutionEnvironment execEnvRead = StreamExecutionEnvironment.getExecutionEnvironment();
-StreamTableEnvironment tableEnv = TableEnvironment.getTableEnvironment(execEnvRead);
+// Old Planner
+StreamTableEnvironment tableEnv = StreamTableEnvironment.create(execEnvRead);
+// Blink Planner, this is recommended.
+StreamTableEnvironment tableEnv = StreamTableEnvironment.create(execEnvRead,
+                EnvironmentSettings.newInstance()
+                        // watermark is only supported in blink planner
+                        .useBlinkPlanner()
+                        .inStreamingMode()
+                        .build());
+tableEnv.connect(pravega)
+                .withFormat(new Json().failOnMissingField(true))
+                .withSchema(schema)
+                .inAppendMode()
+                .registerTableSource("MyTableRow");

```

Comment:
We should add some comment here.
In Flink 1.10, `createTemporaryTable` can be used here as well unless time attributes are added into the schema. (ref. https://issues.apache.org/jira/browse/FLINK-16160)